specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlServer.json

```diff
@@ -0,0 +1,1975 @@
+          "Create or Update a workspace managed sql server's vulnerability assessment with minimal parameters, when storageContainerSasKey is specified": {
+            "$ref": "./examples/PutWorkspaceMangedSqlServerVulnerabilityAssessmentCreateContainerSasKeyMin.json"
+          },
+          "Create or Update a workspace managed sql server's vulnerability assessment with minimal parameters, when storageAccountAccessKey is specified": {
+            "$ref": "./examples/PutWorkspaceMangedSqlServerVulnerabilityAssessmentCreateStorageAccessKeyMin.json"
+          },
+          "Create or Update a workspace managed sql server's vulnerability assessment with all parameters": {
+            "$ref": "./examples/PutWorkspaceManagedSqlServerVulnerabilityAssessmentWithAllParameters.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "WorkspaceManagedSqlServerVulnerabilityAssessments_Delete",
+        "summary": "Remove server's vulnerability assessment.",
+        "description": "Remove workspace managed sql server's vulnerability assessment.",
+        "tags": [
+          "WorkspaceManagedSqlServer, ServerVulnerabilityAssessments"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "vulnerabilityAssessmentName",
+            "in": "path",
+            "description": "The name of the vulnerability assessment.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "VulnerabilityAssessmentName",
+              "modelAsString": true
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully removed vulnerability assessment."
+          },
+          "204": {
+            "description": "Specified vulnerability assessment policy does not exists."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidStorageAccountName - The provided storage account is not valid or does not exist.\n\n * 400 InvalidStorageAccountCredentials - The provided storage account shared access signature or account storage key is not valid.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.\n\n * 500 InternalServerError - Something went wrong."
```

Comment:
are these codes valid for the delete and get? I would assume they should be validated at put time, and not delete or get time

Reply:
This is as per SQL Server swagger and doc. https://docs.microsoft.com/en-us/rest/api/sql/databasevulnerabilityassessments/delete